brainscore/utils/v1_properties_utils.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
```

Comment:
the functions in this file I think should go into either `metrics/distribution_similarity.py` or `benchmarks/{_properties_common.py, marques2020_ringach2002.py}` since they're very specific to these benchmarks

Reply:
They could be used by other benchmarks than the Ringach so they should go to properties_common. But if I put every property calculation method there, that file will become huge.

Reply:
Moved to _properties_common